realm/realm-library/src/androidTest/kotlin/io/realm/entities/MixedIndexed.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.entities
+
+import io.realm.Mixed
+import io.realm.RealmObject
+
+open class MixedIndexed : RealmObject() {
+    companion object {
+        const val FIELD_MIXED = "mixed"
+    }
+
+    // FIXME: Index disabled until https://jira.mongodb.org/browse/RCORE-434 is fixed
```

Comment:
It seems it's fixed now: https://github.com/realm/realm-core/issues/4342

Reply:
Thanks for the heads up!